server/service/src/stock_line/historical_stock.rs

```diff
@@ -63,7 +63,7 @@ pub fn get_historical_stock_lines_available_quantity(
         min_available_and_pack_size
             .entry(stock_line_id)
             .and_modify(|m| {
-                m.total -= quantity;
+                m.total = m.total - quantity;
```

Comment:
😢

Reply:
Lol, you want the short one back???

Reply:
IMHO
 `m.total -= quantity;` looks too much like ` m.total = quantity;`
I had to look twice which made me worry for readability but obviously both are fine!

Reply:
I agree both are fine so I'm leaving it as is 😁